elements/pf-progress/pf-progress.ts

```diff
@@ -129,6 +128,12 @@ export class PfProgress extends LitElement {
   @property()
     variant: '' | 'success' | 'danger' | 'warning' = '';
 
+  #mo = new MutationObserver(() => this.#onMutation());
```

Comment:
We don't need a mutation observer for this we can use `@property` and `willUpdate`